.golangci.yml

```diff
@@ -1,21 +1,21 @@
 # This file configures github.com/golangci/golangci-lint.
 
 run:
+  go: '1.18'
   timeout: 3m
   tests: true
   # default is true. Enables skipping of directories:
   #   vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
   skip-dirs-use-default: true
 
 service:
-  golangci-lint-version: 1.47.3
+  golangci-lint-version: 1.50.1
```

Comment:
If i recall, @zivkovicmilos pointed out pinning to `1.47.3` will help avoid experiencing https://github.com/golangci/golangci-lint-action/issues/535

Reply:
I believe we don't use 1.19.

Reply:
@epikichi is right

It's a linter problem (specifically `gofmt`), that pulls rules for `go 1.19`, regardless of the code go version.

To mitigate this problem, we must pin the `golangci-lint` version to `1.47.3`, because at that version the bundled `gofmt` linter didn't pull rules for `go 1.19`.

Of course, we can unpin this version to `latest`, once we bump the `go-ibft` version to `go 1.19`

Reply:
ok then, will make a task to use 1.19 and block this PR )

Reply:
@zivkovicmilos and now we have it https://github.com/0xPolygon/go-ibft/pull/41#event-7760048031

Reply:
Then you can bump the version to `latest`, so it always pulls in the latest version of the golangci-lint linters

Reply:
it won't work with installing local version `install-deps` so we might have different versions on CI and on developer's machine.